src/App.js

```diff
@@ -1,81 +1,57 @@
 import './App.css';
-import { nanoid } from 'nanoid';
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 import List from './components/List';
-
-const getLocalStorage = () => {
-  let list = localStorage.getItem("list")
-  if (list) {
-    return (list = JSON.parse(localStorage.getItem("list")))
-  } else {
-    return []
-  }
-}
+import { useDispatch } from 'react-redux';
+import { useSelector } from 'react-redux';
+import { addTodo } from './redux/reducers/todoSlice';
+import { updateTodo } from './redux/reducers/todoSlice';
+import { deleteTodo } from './redux/reducers/todoSlice';
```

Comment:
You can import these functions from one import.
```suggestion
import { addTodo, updateTodo, deleteTodo } from './redux/reducers/todoSlice';
```